detections/network/inactive_account_have_activity_detected.yml

```diff
@@ -0,0 +1,80 @@
+name: Inactive Account Have Activity Detected
```

Comment:
can we update this name and the file name to - Detect Network Traffic From Inactive Accounts